FBSnapshotTestCaseDemo/Podfile

```diff
@@ -12,4 +12,12 @@ post_install do |installer|
       config.build_settings['SWIFT_VERSION'] = 4.0
     end
   end
+
+  installer.generated_projects.each do |project|
+      project.targets.each do |target|
+          target.build_configurations.each do |config|
+              config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '14.0'
```

Comment:
as of this: https://github.com/CocoaPods/CocoaPods/issues/4859 pods don't automatically inherit the minimum deployment target, Specta fails to build with missing frameworks if too low but probably could be lower than 14